src/python/sortingview/gui/commonComponents/SelectUnitsWidget/SelectUnitsWidget.tsx

```diff
@@ -8,17 +8,18 @@ type Props = {
     selection: SortingSelection
     selectionDispatch: SortingSelectionDispatch
     curation: SortingCuration
+    sortingSelector?: string
```

Comment:
The sortingSelector is a string that is either "A" or "B", representing first or second sorting in the comparison, respectively.

Reply:
or it could be an empty string, meaning we are not in the comparison view.